less/mixins.less

```diff
@@ -1 +1,15 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.button-green-style() {
+    width: 200px;
+    background-color: @eastern-blue;
+    padding: 14px 0px;
+    text-align: center;
+    color: white;
+    font-size: 2rem;
+    font-family: 'Roboto', cursive;
+    border-radius: 16px;
+    @media @mobile {
+        padding: 3% 5%;
+        justify-content: center;
```

Comment:
In the mobile design file of the site, the buttons take up the width of the container, whereas yours are 200px long and justified left. Maybe you can add something in this selector to better match the design file.

Also, try making this parametric!